catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -841,10 +841,11 @@ private WebClient newOpenSearchClient(String url, Subject subj)
             throws SecurityServiceException {
         SecureCxfClientFactory tempFactory = tempFactory(url);
         WebClient client;
-        if (StringUtils.isNotEmpty(username) && StringUtils.isNotEmpty(password)) {
-            client = tempFactory.getWebClientForBasicAuth(username, password);
-        } else if (subj != null) {
+
+        if (subj != null) {
```

Comment:
I'm not sure this is correct. AFAIK the other ordering was deliberate. There are cases where you may get a subject, but still need to use the user/pass configured for that source, because the subject you have will not be accepted by the remote endpoint.